### PR TITLE
Updates and bug fixes to admin_index page

### PR DESCRIPTION
**Pull request by houseoftech**
I added some more detailed information for tasks which are not yet completed and fixed the reset method.

### CODE REVIEW

File: Controller/QueueController.php [hunk 1]
```diff
@@ -43,11 +44,11 @@ public function admin_reset() {
 		$this->request->allowMethod('post');
 		$res = $this->QueuedTask->truncate();
 		if ($res) {
-			$this->Flash->message(__d('queue', 'OK'), 'success');
```
Comment (dereuromark) [inline Controller/QueueController.php]: The plugin uses the Tools functionality. Is it possible to check if the component(s) exist and conditionally use the default syntax only if not?

**Comment by houseoftech** [comment]
I'm not sure if this is the best method. It seems to work though.

**Comment by dereuromark** [comment]
`isset($this->Flash)` would have worked, probably.

**Comment by houseoftech** [comment]
I tried that and it isn't set until actually used. Becaue of lazy loading, I guess?

**Comment by dereuromark** [comment]
No, if you dont use it in your code it won't ever load.
But if someone uses this via AppControllers $components array it will be available
This the isset() checks works here.

**Comment by houseoftech** [comment]
Okay, I see. That's much cleaner.

**Pull request merged**